symphony-bdk-examples/simple-hello-world/src/main/java/com/symphony/bdk/examples/HelloWorldMain.java

```diff
@@ -0,0 +1,64 @@
+package com.symphony.bdk.examples;
+
+import com.symphony.bdk.core.api.invoker.ApiClient;
+import com.symphony.bdk.core.api.invoker.jersey2.ApiClientJersey2;
+import com.symphony.bdk.core.auth.JwtHelper;
+import com.symphony.bdk.gen.api.AuthenticationApi;
+import com.symphony.bdk.gen.api.MessageApi;
+import com.symphony.bdk.gen.api.MessagesApi;
+import com.symphony.bdk.gen.api.model.AuthenticateRequest;
+import com.symphony.bdk.gen.api.model.V4Message;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.PrivateKey;
+
+@Slf4j
+public class HelloWorldMain {
+
+  private static final String POD_BASE_URL = "https://devx1.symphony.com";
+
+  public static void main(String[] args) throws Exception {
```

Comment:
in the SDK do we provide an Java API close to the REST API or do we provide a Java API that hides a lot of the complexity?
with the second version for instance I would expect to be able to initialize the SDK with auth info and then to perform queries without passing any auth tokens, all of that being taken care internally

Reply:
Yes of course the SDK will provide services that wrap this complexity. We just did not started building them yet :)

Reply:
I created this sample class mainly to confirm that the code generation was working well